src/main/groovy/de/dkfz/eilslabs/batcheuphoria/IntegrationTestStarter.groovy

```diff
@@ -21,71 +22,95 @@ import de.dkfz.roddy.tools.BufferValue
+        Job testJobChild2 = new Job("batchEuphoriaTestJob_Child2", batchEuphoriaTestScript, null, null, new ResourceSet(ResourceSetSize.s, new BufferValue(10, BufferUnit.m), 1, 1, new TimeUnit("m"), null, null, null), null, ["a": "value"], [testParent, testJobChild1], null, jobManager)
+        multipleJobsTest(jobManager,[testParent,testJobChild1,testJobChild2])
+
+        log.severe("Did not test jobManager.queryExtendedJobState")
+        log.severe("Did not test jobManager.waitForJobsToFinish")
+    }
 
 
+    private static void singleJobTest(JobManager jobManager, Job testJob){
+        int maxSleep = 5
+        try {
             log.always("Starting tests for single jobs.")
-            Job testJob = new Job("batchEuphoriaTestJob", batchEuphoriaTestScript, null, new ResourceSet(ResourceSetSize.s, new BufferValue(10, BufferUnit.m), 1, 1, new TimeUnit("20s"), null, null, null), null, ["a": "value"], null, null, jobManager)
 
-            // run single job and check status
-            JobResult jr = jobManager.runJob(testJob)
-            log.postAlwaysInfo("Started ${jr.jobID.id}")
             def jobList = [testJob]
-            ensureProperJobStates(maxSleep, jobList, [jobManager.isHoldJobsEnabled() ? JobState.HOLD : JobState.QUEUED], jobManager)
-
-            log.always("Start and check status")
-            jobManager.startHeldJobs(jobList)
-            // Wait for some seconds and see, if the status changes from HOLD to queued or running and from queued to running
-            // The queued to running check can take a lot more time. Also the default update time for queries to the job system
-            // is too long for tests. We force updates everytime we run queryJobStatus
-            ensureProperJobStates(maxSleep, jobList, [JobState.QUEUED, JobState.RUNNING], jobManager)
 
+            // run single job and check status
+            if(jobManager.isHoldJobsEnabled()) {
+                JobResult jr = jobManager.runJob(testJob)
+                log.postAlwaysInfo("Started ${jr.jobID.id}")
+                ensureProperJobStates(maxSleep, jobList, [JobState.HOLD], jobManager)
+
+                log.always("Start and check status")
+                jobManager.startHeldJobs(jobList)
+                // Wait for some seconds and see, if the status changes from HOLD to queued or running and from queued to running
+                // The queued to running check can take a lot more time. Also the default update time for queries to the job system
+                // is too long for tests. We force updates everytime we run queryJobStatus
+                ensureProperJobStates(maxSleep, jobList, [JobState.QUEUED, JobState.RUNNING], jobManager)
+            } else {
+                JobResult jr = jobManager.runJob(testJob)
+                log.postAlwaysInfo("Started ${jr.jobID.id}")
+                ensureProperJobStates(maxSleep, jobList, [JobState.QUEUED, JobState.RUNNING], jobManager)
```

Comment:
For non-holdEnabled, proper states are: QUEUED, HOLD, RUNNING,(it could even be COMPLETED, depends how fast the check is, but leave C. out here!)

The start is the same for both branches, maybe pull it up.